src/go/k8s/main.go

```diff
@@ -189,11 +193,10 @@ func main() {
 		storageAdvAddr = redpandacontrollers.DetermineAdvStorageAddr(storageAddr, setupLog)
 		storage := redpandacontrollers.MustInitStorage("/tmp", storageAdvAddr, 60*time.Second, 2, setupLog)
 
-		metricsH := helper.MustMakeMetrics(mgr)
+		// TODO this helper has gone away
+		// metricsH := helper.MustMakeMetrics(mgr)
 
-		// TODO fill this in with options
 		helmOpts := helmController.HelmReleaseReconcilerOptions{
-			MaxConcurrentReconciles:   1,                // "The number of concurrent HelmRelease reconciles."
```

Comment:
This is now setup through controller-runtime itself: https://github.com/fluxcd/source-controller/blob/v1.1.0/main.go#L373-L376

Reply:
:nod: